tests/robot-tests/tests/admin_and_public/bau/prerelease_visibility.robot

```diff
@@ -173,22 +150,10 @@ Go to release from admin dashboard
 
 Approve release for immediate publication
     [Tags]  HappyPath
+    # probably should move Sign off into the keyword below it...
```

Comment:
I don't mind `user clicks link Sign off` being outside - makes it clear what's going on.